mimic/rest/ironic_api.py

```diff
@@ -21,6 +21,20 @@ def __init__(self, core):
         """
         self.core = core
 
+    @app.route('/nodes', methods=['POST'])
+    def create_node(self, request):
+        """
+        Responds with response code 201 and returns the newly created node.
+        """
+        return self.core.ironic_node_store.create_node(request)
+
+    @app.route('/nodes/<string:node_id>', methods=['DELETE'])
+    def delete_node(self, request, node_id):
+        """
+        Responds with response code 204 and delete the node.
+        """
+        return self.core.ironic_node_store.delete_node(request, node_id)
```

Comment:
I'm a little concerned about this organization. I feel the `IronicNodeStore` should not deal in HTTP requests and response codes - this smells strongly of inappropriate layer mixing to me.